File: Platform/codesignal/Intro/15.AddBorder/solution.py
def solution(a):
  a= [f'*{x}*' for x in a ]
  star = '*' * len(a[0])
  a.insert(0, star)
  a.append(star)
  return a

print(solution(["abc", "ded"])) # ["*****", "*abc*", "*ded*", "*****"]

File: Platform/codesignal/Intro/15.AddBorder/solution.js
// function solution(p) {
//     const a = p.map((v) => {
//         return `*${v}*`;
//     });
//     const b = '*'.repeat(a[0].length);
//     return [b, ...a, b];
// }

function solution(p) {
  const a = p.map(x => `*${x}*`)
  const b = '*'.repeat(a[0].length);
  return [b, ...a, b]
}

console.log(solution(["abc", "ded"]));